packages/medusa/src/loaders/index.ts

```diff
@@ -92,13 +93,13 @@ export default async ({
 
   await redisLoader({ container, configModule, logger: Logger })
 
-  const modelsActivity = Logger.activity("Initializing models")
+  const modelsActivity = Logger.activity(`Initializing models${EOL}`)
```

Comment:
@srindom I couldn't stand the messy log, so I added EOL that uses the correct operating-system specific new line

Reply:
you don't have to import it first?

Reply:
It is imported [here](https://github.com/medusajs/medusa/blob/eafffdfc8cf50bd163febed3b104c52d0f4d1923/packages/medusa/src/loaders/index.ts#L11).